aten/src/ATen/native/quantized/cpu/q_adaavgpool.cpp

```diff
@@ -40,15 +40,15 @@ template <typename scalar_t>
 static void adaptive_avg_pool2d_single_out_frame(
     scalar_t* input_p,
     scalar_t* output_p,
-    int64_t sizeD,
+    int64_t sizeC,
```

Comment:
Wonder the reason for changing the naming? Is it a convention? `C` means channel, and `D` is instead used for depth in the 3D pooling?

Reply:
For 2d, we should have B: batch, C: channel. I think D is for 3-d case.